src/VolleyManagement.WebClient/spa/app/Components/Schedule/schedule.component.ts

```diff
@@ -48,13 +48,17 @@ export class ScheduleComponent implements OnInit {
     getDivisionsHeaderText(divisionHeader: DivisionHeader): string {
         if (divisionHeader.Id === DummyDivisionHeader.DummyHeaderId) {
             return '\u00A0';//$nbsp to preserve space
+        } else if (divisionHeader.Id === DummyDivisionHeader.PlayOffHeaderId) {
+            return `${divisionHeader.Rounds.join()}`;
```

Comment:
What is the point of interpolation here? simply use divisionHeader.Rounds.join()

Reply:
+